src/selectors.js

```diff
@@ -94,14 +95,7 @@ const createGetP = (polyglotOptions) => {
                     tm: identity,
                 };
             }
-            return {
-                ...p,
-                t,
-                tc,
-                tt,
-                tu,
-                tm,
-            };
+            return assign(p, t, tc, tt, tu, tm);
```

Comment:
from https://lodash.com/docs/4.17.5#assign : 
> This method mutates object

You need to `assign({}, ...)`.
Anyway, `t`, `tc`, `tt`, `tu` and `tm` are functions.

The correct implementation is next : 
```js
return assign({}, p, t, tc, tt, tu, tm);
```